virt/learn_more_about_openshift_virtualization.adoc

```diff
@@ -0,0 +1,108 @@
+| xref:../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[Configuring CSI volumes]
+|
+
+| xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[Learn about virtual machine live migration]
+|
+| Installing {VirtProductName} using the xref:../virt/install/installing-virt-web.adoc#installing-virt-web[{VirtProductName} console] or xref:../virt/install/installing-virt-cli.adoc#installing-virt-cli[CLI]
+|
+
+| xref:../virt/node_maintenance/virt-about-node-maintenance.adoc#virt-about-node-maintenance[Learn about node maintenance]
+|
+|
+|
+
+|
+|===
+
+[id="virtualizaion-administrator"]
+== Virtualization Administrator (Deployment)
+
+[options="header",cols="4*"]
+|===
+|Learn about {VirtProductName} |Deploy {VirtProductName} |Manage {VirtProductName}|Use {VirtProductName}
+
+| xref:../virt/about-virt.adoc#about-virt[Learn about {VirtProductName}]
+| Connecting virtual machines to the xref:../virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc#virt-using-the-default-pod-network-with-virt[default pod network for virtual machines] and xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#attaching-to-multiple-networks[multiple networks]
+| xref:../virt/install/virt-installing-virtctl.adoc#virt-installing-virtctl[Installing the `virtctl` client]
+| xref:../migration-toolkit-for-containers/about-mtc.adoc#about-mtc[Importing virtual machines with the Migration Toolkit for containers]
+
+| xref:../virt/virtual_machines/virtual_disks/virt-features-for-storage.adoc#virt-features-for-storage[Learn about storage features for virtual machine disks]
+| xref:../virt/virtual_machines/virtual_disks/virt-storage-defaults-for-datavolumes.adoc#virt-storage-defaults-for-datavolumes[Configuring data volume defaults]
+| xref:../virt/virt-using-the-cli-tools.adoc#virt-using-the-cli-tools[Using  the CLI tools]
+| xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[Using live migration]
+
+|
+| xref:../virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc#virt-creating-and-using-boot-sources[Creating boot sources and attaching them to templates]
+| Viewing xref:../virt/logging_events_monitoring/virt-logs.adoc#virt-logs[logs] and xref:../virt/logging_events_monitoring/virt-events.adoc#virt-events[events]
+|
+
+|
+| link:https://www.google.com/url?q=https://www.openshift.com/blog/updating-a-boot-source-image&sa=D&source=editors&ust=1626893484915000&usg=AOvVaw0cDIQr63zScl7b-Cn8G0rA[Updating boot source templates]
+| xref:../virt/logging_events_monitoring/virt-monitoring-vm-health.adoc#virt-monitoring-vm-health[Monitoring virtual machine health]
+|
+
+|
+|===
+
+[id="Developer"]
+== Virtual Machine Administrator / Developer (Virtual Machine Consumer)
+
+[options="header",cols="4*"]
+|===
+|Learn about {VirtProductName} |Use {VirtProductName} |Manage {VirtProductName}|Additional resources
```

Comment:
see prior comment about tightening up the table headers.

Reply:
Done.